app/frontend/src/cljs/teet/project/project_view.cljs

```diff
@@ -142,6 +137,64 @@
       (when (:geometry-range? map-settings)
         [search-area-view/feature-search-area e! app project related-entity-type])]]))
 
+(defn- project-timeline [{:thk.project/keys [estimated-start-date estimated-end-date lifecycles]
```

Comment:
Should this project timeline be in it's own NS? like teet.ui.project-timeline or teet.project.project-timeline-view ?

Reply:
Yes. That's a quick change so I'll do it.

Reply:
The item-type -related code should also be ripped from the generic timeline code into the project specific one.